src/Modules/mpoly-localizations.jl

```diff
@@ -176,4 +176,19 @@ the shift map ``Φ : R → R`` which is moving the point of ``𝔪`` to the orig
   return result, a, b
 end
 
+@attr function shifted_module(
+    M::SubModuleOfFreeModule{T}
+  ) where {T<:MPolyLocRingElem{<:Field, <:FieldElem, <:MPolyRing, <:MPolyRingElem,
+                               <:MPolyComplementOfKPointIdeal}}
 
+  R = base_ring(M)::MPolyLocRing
+  P = base_ring(R)::MPolyRing
+  FL = ambient_free_module(M)
+  F = base_ring_module(FL)
+  (A,D) = clear_denominators(M.matrix)
```

Comment:
If it were available for SubModuleOfFreeModule, I would gladly use it. Helas, it only exists for free modules and subquos.

Reply:
Right. I never worked with `SubModuleOfFreeModule`. Should we maybe introduce a getter here? It might not be safe to access the field as it might not even be set (could be used for caching only). 

@jankoboehm : How should we proceed here?